products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -220,6 +220,23 @@ public enum CommandResult {
   }
 }
 
+/// Result of a command execution.
+public struct CommandExtendedResult {
+  public let result : CommandResult /// The result of a command execution
```

Comment:
By convention, Swift has no `:` after the variable name, so `let result: CommandResult`